charts/karmada-operator/crds/operator.karmada.io_karmadas.yaml

```diff
@@ -663,7 +663,7 @@ spec:
                           used. Incorrect settings on this field maybe lead to the
                           corresponding component in an unhealthy state. Before you
                           do it, please confirm that you understand the risks of this
-                          configuration. \n For supported flags, please see https://github.com/karmada-io/karmada/blob/master/cmd/aggregated-apiserver/app/options/options.go
+                          configuration. \n For supported flags, please see https://karmada.io/docs/reference/components/aggregated-apiserver
```

Comment:
The link is dead. should be:
https://karmada.io/docs/reference/components/karmada-aggregated-apiserver